src/tcgwars/logic/impl/gen4/MysteriousTreasures.groovy

```diff
@@ -561,9 +562,32 @@ public enum MysteriousTreasures implements LogicCardInfo {
           resistance F, MINUS20
           pokeBody "Dark Genes", {
             text "As long as Honchkrow has the Energy necessary to use its attack, each of your Murkrow can use Honchkrow’s attack as its own without the Energy necessary to use that attack."
-            delayedA {
-              //TODO
-              //Future TODO: This should work even when Honchkrow levels up, as it has the attack and Poké-Power of the base one (and Lv.X's attacks should also be available to be copied)
+            def selfOwner = self.owner
+            def selfPcs = self
+            getterA GET_MOVE_LIST, { h ->
+              PokemonCardSet hTarget = h.effect.target
+              if (hTarget.active && hTarget.owner == selfOwner && hTarget.name == "Murkrow" && selfPcs.benched) {
+                List<Move> honchkrowMoves = bg.em().activateGetter(new GetMoveList(selfPcs))
```

Comment:
Is this necessary? The body mentions "Honchkrow’s attack" and this should only mean "Dark Wing Flaps" therefore instead of this getter, using `thisCard.moves` would be safer to avoid any potential infinite loops with some weird combo.

Reply:
If Honchkrow is leveled up, it should obtain the attacks from both its base form and the Level X. I believe originally this code took the attacks that way, if you think it'll avoid corruption issues it can be reverted.

Reply:
Ah, I am not used to Level Xs yet :D Well in this case I believe [a recursion guard like this](https://github.com/axpendix/tcgone-engine-contrib/blob/master/src/tcgwars/logic/impl/gen4/LegendsAwakened.groovy#L1048) could be useful here.